R/gg-plots.R

```diff
@@ -320,25 +388,26 @@ ggally_cor <- function(
     }
 
     if (any(!rows)) {
+      if (!is.null(colorData) && (length(colorData) == length(xData))) {
+        colorData <- colorData[rows]
+      }
+      xData <- xData[rows]
+      yData <- yData[rows]
+
       total <- sum(!rows)
       if (total > 1) {
         warning("Removed ", total, " rows containing missing values")
```

Comment:
Should it be a simple message instead of a warning?

Reply:
I remember trying to copy ggplot2.  Have they moved it to a message?

Reply:
In ggplot2, (see for example `stat_count()`), NA are always removed and there is a warning except if the user said explicitely `na.rm=TRUE`.

Here, NA are removed only then explictly said by the user. So it is expected and should not result in a warning.

Or did I miss something?

Reply:
Great catch!!

I'll change it to `na.rm = NA`.  If `is.na(na.rm))`, then I'll produce the warning.  `na.rm = NA` will behave like `na.rm = TRUE`.

As you said above, there is not need to have a `complete.cases`.